cortex/run.py

```diff
@@ -195,6 +203,24 @@ def get_first_last_datapoint(participant, original_time, resolution, start = 1):
     limit_value = 1
     if start:
         limit_value = -1
+
+    func_list = {f['callable'].__name__: f for f in all_features()}
+    dependent_feats = [f for f in func_list[feature]["dependencies"]]
+    
+    primary_mods = [mod_name for mod_name, mod in inspect.getmembers(primary, inspect.ismodule)]
+    
+    dependent_feats_raw=[]
+    for f in dependent_feats:
+        edited=f
+        if 'lamp' in f:
```

Comment:
When I run this, I get an error here. `dependent_feats` is a list of functions. @carlan1